gradio/ipython_ext.py

```diff
@@ -0,0 +1,75 @@
+try:
+    from IPython.core.magic import register_cell_magic
+except ImportError:
+    pass
+
+import gradio
+
+
+def load_ipython_extension(ipython):
+    demo = gradio.Blocks()
+
+    @register_cell_magic
+    def blocks(line, cell):
+        if "imp" in line:
+            from gradio.components import (
```

Comment:
why do we have to import everything like this? is there a more maintainable way?

Reply:
>Can we import everything from init.py?

Hmm I think that will create a circular import

But I'll use wildcards, I think that's pretty maintainable

Reply:
Actually nvm: 
<img width="562" alt="image" src="https://user-images.githubusercontent.com/1778297/169163335-36b766ed-3d8e-458f-9235-0f864e105e0e.png">